django/contrib/staticfiles/finders.py

```diff
@@ -21,6 +24,17 @@ class BaseFinder(object):
     """
     A base file finder to be used for custom staticfiles finder classes.
     """
+    def check(self, **kwargs):
+        """
```

Comment:
@timgraham  None of other `check()` methods are with docstring. I think I'll just remove the rest.